app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md

```diff
@@ -0,0 +1,146 @@
+* Removing the hint text suggesting candidates add the name of their own company if self-employed – this didn’t seem to be required.
+* Changing the end date from “End date (leave blank if this is your current role)” to “Are you still working in this job?”. If the candidate answered “No” then a ”When did you leave this job?” question was revealed.
+* Changing the start date from “Start date” to “When did you start this job?”, for consistency with the “When did you leave this job?” question, and to better reflect the fact that we don’t ask for a full date, only month and year.
+
+When a candidate added a job, they were returned to the Work history review page, rather than being asked whether or not they wanted to add another job. This was partly to solve an issue seen in production where candidates go to add another job, but then leave all the fields blank and submit the form, causing validation errors.
+
+Our hypothesis was that this is because they either changed their mind or didn’t realise that they would need to add another job immediately. Additionally, by taking candidates back to the review page after adding each job, they are able to review their progress in adding jobs so far, and can remind themselves where they’ve got to – which may be more important when adding lots and lots of jobs.
+
+We redesigned the review page. Instead of using summary cards for each job, the jobs were presented in a table format. This was intended to remove the duplication caused by repeating the field names in each summary card, and make it easier for candidates to scan the page, as each job takes up less vertical height.
+
+## Results of the user research
+
+We tested a prototype with XX candidates. We recruited older candidates so that we could test the new section with people who likely had a longer work history.
+
+We also tested a mock-up of applications showing long work histories with providers, to check our assumptions.
+
+We found that moving the work history section beneath the Safeguarding heading did not communicate to candidates why we were asking for this information. If anything, they assumed that we might be asking for experience of safeguarding within their work (in part because many of them had some awareness of safeguarding from working in schools or with children).
+
+Candidates were able to add lots of jobs fairly quickly - likely much more quickly than if they were still being asked for skills and experience relevant to teaching for each job.
+
+Some candidates were unclear that they needed to add their complete work history, either not reading or ignoring the guidance. Instead they only added 'relevant' or 'proper' jobs, and omitting their first jobs working in bars or in retail.
+
+Some candidates expressed surprise that they weren’t asked for more details. Whilst some were happy with how quick it was, others wanted to be able to explain more about their roles to support their application.
```

Comment:
You say ‘some’ a lot. Would be good to get numbers out of total numbers here and all the other places where you said ‘some’

Reply:
Good point - I'm hoping the research team can help flesh out this section a bit more.

Reply:
Numbers are relevant when you're estimating how prevalent an issue is in a user population –– they're not so useful to  with very small sample because they usually tell us very little about how often it would actually happen in real life. 

The important point is that we discovered and fixed issues before real users had to deal with them! (Not how many in the small sample reported that issue.)